src/languageservice/parser/jsonParser07.ts

```diff
@@ -1066,6 +1066,21 @@ function validate(
             }
           }
           break;
+        case 'ipv4':
```

Comment:
What is the reason why `ipv4` and `ipv6` are not added to the `formats` and handled the same way above? I think it is better to unify the implementation path which also fixes how the message should be handled.